tests/unit/amo/test_utils.js

```diff
@@ -22,4 +22,24 @@ describe('amo/utils', () => {
       expect(getErrorComponent(501)).toEqual(ServerError);
     });
   });
+
+  describe('getDjangoBase62', () => {
+    const base62 = getDjangoBase62();
+
+    it('encodes a example number', () => {
+      expect(base62.encode(535493287492)).toEqual('9QVvGFw');
```

Comment:
As this test uses the same values as the decode test, it might be worth declaring these two values as consts, and then we can be sure that we are using the same values going in both directions and avoid typo related issues.

Reply:
Good idea 👍